doc/api/fs.md

```diff
@@ -4929,6 +4929,20 @@ will be passed as `Buffer` objects.
 If `options.withFileTypes` is set to `true`, the resolved array will contain
 [`fs.Dirent`][] objects.
 
+Example iterating over a directory listing:
```

Comment:
You can probably omit this line and simply show the example code. If you want to include `// Iterate over a directory listing.` as a comment in the example code, that would be better IMO. The example code and surrounding text should be related but independent.